build.sbt

```diff
@@ -70,8 +70,7 @@ lazy val runtime = CrossProject(
     crossTarget := target.value / s"scala-${scalaVersion.value}",
     crossVersion := CrossVersion.full,
     libraryDependencies ++= Seq(
-      "org.scalatest" %%% "scalatest" % scalatestVersion % Test,
-      "org.scala-lang" % "scala-compiler" % scalaVersion.value % Compile
```

Comment:
@lefou you were totally correct... this doesn't need to be a dependency for the runtime. If you look back at the commits it seems to have slipped in when I refactored the build definition.

Reply:
What I wonder is, whether we can avoid the `CrossVersion.full` for the runtime.

Reply:
So from digging further in, from what I can tell, no this doesn't need a full cross version. I've gone ahead and tested this locally and also with the sbt plugin to verify. I've gone ahead and removed the `CrossVersion.full` on the runtime and also then could remove the hacky thing we were doing to name the scalaJS artifact correct as well since it's now done correctly automatically.

Reply:
You should be able to give this a try now wtih `1.4.5+1-775fcaca-SNAPSHOT`. Report back if something seems awry. I'd be good to get this more widely tested before a release.